### DISCUSSION TITLE
Make FinancialService.ListFinancialEventsByOrderId(string orderId) public

### DISCUSSION THREAD

**Discussion by IvanMConnex** [abuzuhri/Amazon-SP-API-CSharp#59] [Ideas]
Hello, how are you doing today? My name is Ivan, and I am with Sync with Connex.

We're looking into using your SDK for our Amazon integration, and we wanted to ask: Could you please make FinancialService.ListFinancialEventsByOrderId(string orderId) public? We need it for our use case.

Thank you, and have a good day.

**Comment by abuzuhri** [comment]
hello @IvanMConnex 

please check v1.0.33 in NuGet I fix it .
